### DISCUSSION TITLE
With Google Maps: Uncaught TypeError: Cannot read properties of null (reading 'fromLatLngToDivPixel')

### DISCUSSION THREAD

**Discussion by apalisa** [visgl/deck.gl#8878] [Q&A]
First off, I'm new, I apologise if I'm not too technical or whatever. Ask me whatever you want to know and I will do everything posible to respond to you.

I've tryed a lot of diferent ways to make it work, but I really couldn't. I don't know if it's just me or it is a bug of the library. I've copied the "Example: Use deck.gl with React", and I got this error in the browser's console:
`Uncaught TypeError: Cannot read properties of null (reading 'fromLatLngToDivPixel')`
   ` at getViewPropsFromOverlay (@deck__gl_google-maps.js?v=ac431520:75:31)`
   ` at GoogleMapsOverlay._onDrawRaster (@deck__gl_google-maps.js?v=ac431520:369:51)`
    `at Ena.draw (overlay.js:4:891)`
   ` at Fna.Ri (overlay.js:4:1116)`
    `at zma (map.js:51:469)`
    `at map.js:50:454`
If someone could help me, I would appreciate it.

I'm using React with Vite and I need to use JavaScript Google Maps API.
Browser: Opera One(version: 108.0.5067.29)
OS: Windows 10 Pro
 
Again I apaologise if there is information missing, just ask and I will post it. I'm just new and don't really know what else you need for you to help me.

**Comment by mateussatoh** [comment]
Post the props you are passing to Map, in my case i had to create a custom [mapId](https://developers.google.com/maps/documentation/get-map-id) using the type Vector, and create and associate a [map styles](https://developers.google.com/maps/documentation/javascript/examples/map-id-style) to it.

**Comment by Pessimistress** [comment]
If you mean [this example](https://github.com/visgl/deck.gl/tree/master/examples/get-started/react/google-maps), you need to follow the README and set the map API key and map ID.

**Comment by alanloffler** [comment]
Remove StrictMode tags on main.tsx (same error with React 19.0.0)
